support-files/mysql.spec.sh

```diff
@@ -124,104 +120,42 @@
 %endif
 
 # ----------------------------------------------------------------------------
-# Packager
+# Distribution support
 # ----------------------------------------------------------------------------
-%if %{undefined mysql_packager}
-%define mysql_packager MySQL Build Team <build@mysql.com>
+
+# Disable post build checks for time being.
+BuildConflicts: post-build-checks rpmlint
```

Comment:
I don't like that.  IMO, we should allow checks to be run, and just override those "error" results to whose policies we cannot adapt.
The means for this is to have another source file "mysql-rpmlintrc", specify it as a separate source (like "source99:"), and specify what to disable. In my local tests, these 2 lines worked (comment + rule):
   # allow the traditional name "mysql-shared" even with a so-versioned lib
   setBadness('shlib-policy-name-error', 0)
I just don't know where to put that file so that it remains a separate source - only in OBS?